mod/src/main/java/eu/the5zig/mod/util/Updater.java

```diff
@@ -59,7 +58,7 @@ public void run() {
 			URL url = new URL("https://secure.5zigreborn.eu/version?mc=" + Version.MCVERSION);
 			HttpURLConnection connection = (HttpURLConnection) url.openConnection();
 			connection.addRequestProperty("User-Agent", "5zig/" + Version.VERSION);
-			Download download = The5zigMod.gson.fromJson(IOUtils.toString(connection.getInputStream()), Download.class);
+			Download download = The5zigMod.gson.fromJson(String.valueOf(connection.getInputStream()), Download.class);
```

Comment:
Revert, String.valueOf will call InputStream#toString, which is not what we want.